students/Gabe_Lamberth/lesson02/series.py

```diff
@@ -0,0 +1,66 @@
+# Create Fibonacci Function
+def fibonacci(num):
+    if (num < 0):
+        print("Incorrect input")
+        # First Fibonacci number is 0
+    elif (num == 0):
+        return 0
+        # Second Fibonacci number is 1
+    elif (num == 1):
+        return 1
+    else:
+        return fibonacci(num - 2) + fibonacci(num - 1)
+    
+def lucas_nums(num):
+    if (num == 0):
+        return 2
+    elif (num == 1):
+        return 1
+    else:
+        return lucas_nums(num - 2) + lucas_nums(num - 1)
+    
+def sum_series(reqNum, opt1 = 0, opt2 = 1):
```

Comment:
No camel case in python :) . Stick to underscore to separate words. https://www.python.org/dev/peps/pep-0008/